sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/schema/SchemaBaseBeamTable.java

```diff
@@ -18,14 +18,14 @@
 package org.apache.beam.sdk.extensions.sql.impl.schema;
```

Comment:
Should we also move this to `org.apache.beam.sdk.extensions.sql.meta`?

Reply:
Agreed, will do it in a split PR1.